odoo/tools/mail.py

```diff
@@ -190,6 +190,10 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
     src = src.replace(u'<%', misc.html_escape(u'<%'))
     src = src.replace(u'%>', misc.html_escape(u'%>'))
 
+    # On the specific case of Outlook desktop it adds unnecessary '<o:.*></o:.*>' tags which are parsed in '<p></p>'
+    # and break the sanitization of the mail body
```

Comment:
the comment is a little unclear, I would say the tags are "replaced  by `<p></p>`" and "which may alter the appearance (eg. spacing) of the mail body"

also if possible (even if the other lines doesn't do it), it's better to cut the line at at most 100 characters